tuplex/core/src/TransformStageExceptionReservoir.cc

```diff
@@ -406,6 +406,10 @@ namespace tuplex {
                 // op is valid
                 assert(op);
                 switch(op->type()) {
+                    case LogicalOperatorType::SORT: {
+                        // TODO: COLBY
```

Comment:
remove TODO, add description why this can be skipped here. Should logic here be added?